### DISCUSSION TITLE
CodeWithAloha Onboarding Thread

### DISCUSSION THREAD

**Discussion by tyliec** [CodeWithAloha/docs#4] [General]
# 🌊 👋 Welina Mai / Welcome 👋 🌊

@CodeWithAloha welcomes everyone! Whether you are an expert or beginner, we have a spot for you.

# Summary

Imagine a world where your taxes go to programs you're proud of, people get the help they need, and the public trusts government again. These are some of the problems we face with civic tech. Not only are these problems extremely difficult with an extremely large scope (quite literally, government serves everyone), but public digital infrastructure also has to stand the test of time and be durable for citizens wherever and whenever.

The Alliance of Civic Technologists (ACT) is a newly formed organization that will connect volunteer civic technology efforts, in both technology and advocacy, in their local communities. Our initial goal is to ensure the continuity and growth of civic technology volunteer communities across the United States, launching immediately to fill the gap left by the sunsetting of Code for America’s Brigade program. Learn more about ACT here: https://www.civictechnologists.org/.

For Hawaii Brigade specific information, check out Code With Aloha's GitHub organization (https://github.com/codewithaloha) or our homepage (https://codewithaloha.org/) to see what we've been up to! You can check us out on Meetup: https://www.meetup.com/g/code-with-aloha/ to see our latest events.

# Our Focus

**Meet new people**

Build with, not for. We are a community of people who are passionate about helping others. We are here to support each other and build together.

**Make a Difference**

Nobody is coming to save us. It's up to us to impact our community and drive change. We are the ones we've been waiting for. We are the change that we seek.

# Get Started

As part of your initial onboarding process and to have you start "contributing" to the group ([yay GitHub green squares!](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/viewing-contributions-on-your-profile#what-counts-as-a-contribution)).

**Steps**

1. Get invited to the @CodeWithAloha GitHub organization: https://github.com/codewithaloha. Some people you can ask are either @tyliec @avenmia or @sthapa.
2. Leave an answer on this discussion post answering the following questions:

```
1. What makes you want to join/get involved with Code With Aloha?

Example Answer from Luany L:

I was interested in creating on a project that bridges digital and physical experiences. the hawaii atlas was the perfect opportunity for that!

2. What civic areas are you interested in?

Example Answer from @ggordn3r: Elections, urban planning, land use/zoning, economic development, schools
```

**Comment by kfmfe04** [comment]
1.  Want to help contribute back to the community.
2. Urban planning, schools, land use/zoning

**Comment by mhpoon** [comment]
1. I am interested in learning more about software engineer and also contribute to the Hawaii community. 
2. Economic Development, schools, urban planning

**Comment by k7lim** [comment]
1. I am interested in contributing code reviews and understanding what challenges i can help with
2. I'm interested in supporting journalism, elections and fair housing

**Comment by tyliec** [comment]
Closing this discussion in favor of [Slack's onboarding workflow automation](https://slack.com/help/articles/4412443174547-Automations--Welcome-new-hires-in-Slack) in the `#hello-world` channel - I think that fits the purpose better
